src/cg/cst_cg_map.c

```diff
@@ -77,16 +77,22 @@ cst_cg_db *cst_cg_load_db(cst_voice *vox, cst_file fd)
     cst_cg_db *db = cst_alloc(cst_cg_db, 1);
     int i;
     uint32_t elements[2];
-    uint32_t load_buff[4];
+    struct load_buff_s {
```

Comment:
A good idea.

I was mainly testing my options with this psuh. Frankly I didn't think this would pass Travis.